server/.env

```diff
@@ -0,0 +1,136 @@
+USERNAME= mohamedabdi
```

Comment:
This `.env` file should be in the `.gitignore` file - this has commited secret values to version control